gatsby/create-pages/create-redirects.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Sometimes we mess up URLs. Fixing them is necessary, but if Google or other
+ * Link on external pages still link to the old URL they will only get the 404
+ * page. That is why we need to redirect the old links to the new ones.
+ */
+const createRedirects = ({ actions }) => {
+  const { createRedirect } = actions;
+
+  const redirects = [
```

Comment:
the previous version was this one. Don't we need the `/career/325433-senior-backend-engineer-(mwx)/` variant to be redirected? Or wasn't it indexed before? 
```
const redirects = [
    {
      fromPath: '/career/325433-senior-backend-engineer-m-w-x/',
      toPath: '/career/325433-senior-backend-engineer-(mwx)/',
    },
    {
      fromPath: '/career/325420-senior-frontend-engineer-m-w-x/',
      toPath: '/career/325420-senior-frontend-engineer-(mwx)/',
    },
    {
      fromPath: '/career/338504-frontend-engineer-w-m-x/',
      toPath: '/career/338504-frontend-engineer-(wmx)/',
    },
    {
      fromPath: '/career/325493-senior-uxui-designer-m-w-x/',
      toPath: '/career/325493-senior-uxui-designer-(mwx)/',
    },
    {
      fromPath: '/page/imprint/',
      toPath: '/imprint/',
    },
  ];
```

Reply:
You are right. We also need to redirect to "original" path now. Will add it. Good catch :)